src/CMakeLists.txt

```diff
@@ -362,6 +362,8 @@ if(MINGW)
   set(CPACK_NSIS_MUI_UNWELCOMEFINISHPAGE_BITMAP "${CPACK_NSIS_MUI_WELCOMEFINISHPAGE_BITMAP}")
   set(CPACK_NSIS_CREATE_ICONS_EXTRA "CreateShortCut '$SMPROGRAMS\\\\$STARTMENU_FOLDER\\\\${PROGNAME}.lnk' '$INSTDIR\\\\${PROGNAME}.exe'")
   set(CPACK_NSIS_DELETE_ICONS_EXTRA "Delete '$SMPROGRAMS\\\\$START_MENU\\\\${PROGNAME}.lnk'")
+  set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS "ExecWait 'Taskkill /IM keepassxc-proxy.exe /F'")
+  set(CPACK_NSIS_EXTRA_UNINSTALL_COMMANDS "ExecWait 'Taskkill /IM keepassxc-proxy.exe /F'")
```

Comment:
Need to add keepassxc.exe, same as above

Reply:
Windows is case-insensitive, but I would still use the correct KeePassXC.exe, not keepassxc.exe.